ansible_kind/scripts/multiclusterkind/kind_create_clusters.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Parameters: [HOST_IPADDRESS](optional) if you want to export clusters' API server port to specific IP address
+# 1. used by developer to setup develop environment quickly.
+# 2. used by e2e testing to setup test environment automatically.
+
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")
+# shellcheck source=/dev/null
+source "${REPO_ROOT}"/util.sh
+
+# variable define
+KUBECONFIG_PATH=${KUBECONFIG_PATH:-"${HOME}/.kube"}
+# hardcode the following
+F5GCNI_SETUP="YES"
+SETUP_METALLB="INSTALL_METALLB"
+NUM_CLUSTERS=${1:-4}
+CLUSTER_VERSION=${CLUSTER_VERSION:-"kindest/node:v1.21.1"}
+KIND_LOG_FILE=${KIND_LOG_FILE:-"/tmp/targetClusters"}
+
+# Make sure go exists and the go version is a viable version.
+util::cmd_must_exist "go"
+util::verify_go_version
+
+# install kind and kubectl
+kind_version=v0.11.1
+echo -n "Preparing: 'kind' existence check - "
+if util::cmd_exist kind; then
+  echo "passed"
+else
+  echo "not pass"
+  util::install_kind $kind_version
+fi
+# get arch name and os name in bootstrap
+BS_ARCH=$(go env GOARCH)
+BS_OS=$(go env GOOS)
+# check arch and os name before installing
+util::install_environment_check "${BS_ARCH}" "${BS_OS}"
+echo -n "Preparing: 'kubectl' existence check - "
+if util::cmd_exist kubectl; then
+  echo "passed"
+else
+  echo "not pass"
+  util::install_kubectl "" "${BS_ARCH}" "${BS_OS}"
+fi
+#prepare for kindClusterConfig
+TEMP_PATH=$(mktemp -d)
+echo -e "Preparing kindClusterConfig in path: ${TEMP_PATH}"
+i=1
+while [ "$i" -le "$NUM_CLUSTERS" ];
+do
+  FILE="${TEMP_PATH}"/cluster${i}.yaml	
+cat << EOF > "$FILE"
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  podSubnet: "{{pod_subnet}}"
+  serviceSubnet: "{{service_subnet}}"
+featureGates:
+  EndpointSliceProxying: true
```

Comment:
This feature gate came GA in [1.22](https://v1-22.docs.kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)
```suggestion
```